gui/FunctionPlotterInput.cpp

```diff
@@ -16,10 +16,15 @@ App * FunctionPlotterInput::handleEvent(SDL_Event event) {
 	}
 
 	if(event.type == SDL_KEYDOWN && event.key.keysym.sym == SDLK_RETURN) {
-		this->functionPlotter->makeFunction(this->text);
-		this->graph->setObjects(this->functionPlotter->getFunction());
+		if (this->text == "+") {
```

Comment:
#19 We could add a button or even 2 keys to press without the input field to zoom in and out
If we find time for that 🤷‍♂️

Reply:
Yes, that would be cool. But I doubt we find time for that to be honest.